components/ui/PopoverHint.tsx

```diff
@@ -0,0 +1,101 @@
+'use client';
+
+import { useRef, useEffect } from 'react';
+import { Info, X } from 'lucide-react';
+import { motion, type Variants, useCycle } from 'framer-motion';
+import TooltipHint from './TooltipHint';
+import { useTranslations } from 'next-intl';
+
+export default function PopoverHint({
+  hint,
+  title,
+}: {
+  hint: string;
+  title?: string;
+}) {
+  const [showHint, toggleShowHint] = useCycle(false, true);
+  const hintRef = useRef<HTMLDivElement>(null);
+  const t = useTranslations('PopoverHint');
+
+  // Keyboard shortcut to toggle hint
+  useEffect(() => {
+    const handleKeyPress = (event: KeyboardEvent) => {
+      if (event.key === 'i' && event.metaKey) {
+        // CMD + I
+        toggleShowHint();
+      }
+    };
+
+    document.addEventListener('keydown', handleKeyPress);
+
+    return () => {
+      document.removeEventListener('keydown', handleKeyPress);
+    };
+  }, [toggleShowHint]);
+
+  const variants: Variants = {
+    open: {
+      clipPath: `circle(150% at 90% 0)`,
+      transition: {
+        type: 'spring',
+        stiffness: 100,
+        damping: 20,
+        restDelta: 2,
+      },
+    },
+    closed: {
+      clipPath: 'circle(30px at 90% 0)',
+      transition: {
+        type: 'spring',
+        stiffness: 400,
+        damping: 40,
+      },
+    },
+  };
+
+  return (
+    <div>
+      <TooltipHint
+        hint={t('hint')}
+        additionalContent={tooltipKeyboardShortcut()}
```

Comment:
Would that be a good idea to put the `tooltipKeyboardShortcut()` content in the `hint` prop and make it's type `ReactNode`, what do you think?
Smt like:
```
<TooltipHint
        hint={
          <>
            {t('hint')}
            {tooltipKeyboardShortcut()}
          </>
        }
```
or putting the `tooltipKeyboardShortcut` content directly

Reply:
`additionalContent` prop seems a little unnecessary to me (again ignore this if you think your way is better)

Reply:
Completely agree. Resolved here [2e7f7ba](https://github.com/complexdatacollective/Studio/pull/30/commits/2e7f7ba7894c93ed60a1bf33fae3bb3d7ef05d55)